stressberry/cli.py

```diff
@@ -65,7 +71,22 @@ def _get_parser_run():
         "--cores",
         type=int,
         default=None,
-        help="number of cpu cores to stress (default: all)",
+        help="number of CPU cores to stress (default: all)",
+    )
+    parser.add_argument(
+        "-f", "--frequency", help="measure CPU core frequency", action="store_true"
+    )
+    parser.add_argument(
+        "-ff",
+        "--frequency-file",
+        type=str,
+        default="/sys/devices/system/cpu/cpu0/cpufreq/scaling_cur_freq",
+        help="CPU core frequency file. Must be used in conjunction with --disable-vcgencmd if vcgencmd exists (default: /sys/devices/system/cpu/cpu0/cpufreq/scaling_cur_freq)",
+    )
+    parser.add_argument(
```

Comment:
Yeah I don't know if we need this at all. I'd be fine with just using `vcgencmd` without providing the fallback. Makes the code simpler.